src/components/help/help.pw.tsx

```diff
@@ -293,7 +294,10 @@ test.describe("Accessibility tests for Help component", () => {
         <HelpComponentTest isFocused={boolVal}>{tooltipText}</HelpComponentTest>
       );
 
-      await checkAccessibility(page);
+      await checkAccessibility(
+        page,
+        boolVal ? tooltipPreview(page) : undefined
+      );
```

Comment:
**suggestion**: Given we now have a conditional statement in the test, what do you think about splitting this test in two?

```
test("when isFocused prop is true, tooltip passes accessibility checks", () => { /* ... */ });

test("when isFocused prop is false, component passes accessibility checks", () => { /* ... */ });
```

Reply:
I've gone with `should check when isFocused prop is` as it's already got the describe that talks about accessibility etc